client/src/components/_common/Button/Button.jsx

```diff
@@ -64,37 +76,18 @@ const Button = ({
   }
   /* eslint-enable no-console */
 
-  const buttonRootClass = styles['root'];
-
-  let buttonTypeClass;
-  if (type === 'link') {
-    buttonTypeClass = styles['as-link'];
-  } else if (type === 'primary' || type === 'secondary') {
-    buttonTypeClass = styles[`${type}`];
-  } else if (type === '') {
-    buttonTypeClass = type;
-  }
-
-  let buttonSizeClass;
-  if (size === 'small') {
-    buttonSizeClass = styles['size-small'];
-  } else {
-    buttonSizeClass = styles[`width-${size}`];
-  }
-
-  const buttonLoadingClass = isLoading ? styles['loading'] : '';
-
```

Comment:
**DRY-er:** Now that [type and size maps exist](https://github.com/TACC/Core-Portal/pull/640/files#diff-980357535fcb70c556355c2b7c57390811d8cbae7607ac38c5f718ca0cbd62f6L8-R21), this can be simplified to [logic directly in `className`](https://github.com/TACC/Core-Portal/pull/640/files#diff-980357535fcb70c556355c2b7c57390811d8cbae7607ac38c5f718ca0cbd62f6L90-R85).